### PR TITLE
Validate with bake

### PR DESCRIPTION
**Pull request by crazy-max**

### CODE REVIEW

File: docker-bake.hcl [hunk 1]
```diff
@@ -19,6 +19,27 @@ target "test-debian" {
...
+    targets = ["shfmt-validate", "shellcheck"]
+}
+
+target "shfmt-validate" {
+    dockerfile = "./hack/shfmt.Dockerfile"
+    target = "validate"
+    output = ["type=cacheonly"]
+}
+
+target "shfmt-update" {
```
Comment (tonistiigi) [inline docker-bake.hcl:32]: nit: this can just be called shfmt

**Comment by tonistiigi** [comment]
Why the dockerfile renames? Not that there is a correct solution

**Comment by crazy-max** [comment]
@tonistiigi 

> Why the dockerfile renames? Not that there is a correct solution

Thought it was better to have in root hack folder instead of a subfolder as the shell scripts have been removed but can bring them back there if you want.

File: docker-bake.hcl (hunk 1 above)
Comment (crazy-max) [reply docker-bake.hcl]: Done

**Comment by tonistiigi** [comment]
Normally we don't put dockerfiles in `hack/` as it is used for runnable scripts.

**Comment by crazy-max** [comment]
@tonistiigi 

> Normally we don't put dockerfiles in `hack/` as it is used for runnable scripts.

Ok moved them back

**Review by tonistiigi — APPROVED** [review]

**Pull request merged**